analyzer/txtracer.go

```diff
@@ -69,12 +69,15 @@ func (tr *Tracer) TraceTransaction(blockHeader *types.Header, tx *types.Transact
 
 	// We assume both arrays are in order
 	// then look for the first lockedGold operation with a change in AccMain
-	// and look for the matching transfer
+	// and look for the matching transfer.
+	//
+	// TobinTax transfers for locked gold operations are included in transferOperations
+	// and are not filtered out here, though the duplicate locked gold operations are.
 	ti := 0
 	for _, lgOp := range lockedGoldOperations {
```

Comment:
Won't work, here's the explanation:
```
//For a locked gold operation that got a tobin tax of 10%  you'll have
// lock(100)
// tax 10
// The event will be `GoldLocked(fromAccount, 90)`
// The Transfer Operation
fromAccountMain       -100
lockedGolContractMain   90
tobinRecipientAccount   10
 // LockedGold operation (created from GoldLocked event) will be:
fromAccountMain                 -90
fromAccountLockedNonVoting       90
lockedGolContractMain            90
// Now we need to figure out that both are the SAME operation group, and output
fromAccountMain                 -100
fromAccountLockedNonVoting       90
lockedGolContractMain            90
tobinRecipientAccount            10
```

I would comment tobin tax computation for now, and do it in another pr